src/main/kotlin/com/launchdarkly/intellij/hover/HoverDocumentationProvider.kt

```diff
@@ -88,7 +88,7 @@ class HoverDocumentationProvider : AbstractDocumentationProvider() {
         }
 
         val flag = getFlag(element) ?: return null
-        val flagConfig: FlagConfiguration = getFlagConfig(element, flag.key)
+        val flagConfig: FlagConfiguration = getFlagConfig(element, flag.key) ?: return null
```

Comment:
When we are constructing the view model on line 97 below:

```kotlin
put("on", flagConfig.on) // If flagConfig is null, then this will throw an exception
```

Reply:
If we return null, we will not reach line 97 right?

Reply:
Resolving this, lmk if we need to revisit this point.